packages/create/src/cli.ts

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+import { Command } from 'commander';
+import prompts from 'prompts';
+
+import packageJson from '../package.json' with { type: 'json' };
```

Comment:
Adding a `package.json` export would allow this to be path agnostic (i.e. not relative), e.g. https://github.com/seek-oss/sku/blob/ac06eef4c8061d6b252339093a0d146c82844d4d/packages/sku/package.json#L34

```suggestion
import packageJson from '@sku-lib/create/package.json' with { type: 'json' };
```